smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -128,10 +131,7 @@ contract MixinTransfer is
     public
   {
     _isValidKey(_tokenId);
-    if(
-      // the specified address does not own the token
-      ownerOf(_tokenId) != _from
```

Comment:
Expected?

Reply:
hmm no. I rewrote the conditions to make them more readable